src/main/java/it/nanowar/ofsteel/helloworld/HelloWorldMainLauncherClass.java

```diff
@@ -42,8 +42,13 @@ public static void main (String [] args) {
 	public void falseMethod() {
 		boolean metal=false;
 		if(metal==false) {
-			String joeyDeCaio=null;
-			joeyDeCaio.length();
+			try {	
+				String joeyDeCaio="";
+				joeyDeCaio.length();
+			}
+			catch(Exception ex) {
+				System.out.println(ex);
```

Comment:
If you're at it, it should rather be

```java
System.err.println(ex);
```

but I am unsure they'll need to adjust the song too...